src/main/java/com/ceritoxi/euler2/Euler2.java

```diff
@@ -0,0 +1,21 @@
+package com.ceritoxi.euler2;
+
+import com.ceritoxi.util.Math;
+
+public class Euler2 {
+	
+	private int result;
+	
+	public Euler2() {
+		this.result = 0;
```

Comment:
this line is not needed

Reply:
it's just a habit of mine from C to initialize everything. But I guess I can delete it.